.dockerignore

```diff
@@ -0,0 +1,3 @@
+.git
```

Comment:
Is there a good way to leverage the .gitignore here?

If not, we may want to add `.idea/` and `.project` for common IDEs.

Reply:
Perhaps, but this is about docker. :)

Reply:
The .dockerignore allows docker to ignore specific files that might trigger an image rebuild.  Files listed there cannot be used in a Dockerfile.